realm/realm-library/src/androidTest/java/io/realm/RealmAnnotationTests.java

```diff
@@ -227,6 +228,63 @@ public void primaryKey_isIndexed() {
         assertTrue(table.hasSearchIndex(table.getColumnIndex("id")));
     }
 
+    @Test
```

Comment:
I think so. As long as we covered the corner case related with `RealmObjectSchema.xxxPrimaryKey`, is good enough then IMO.